.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: shell
+dist: xenial
+env:
+  global:
+    - URL=http://cdimage.ubuntu.com/ubuntu-base/xenial/daily/current/xenial-base-amd64.tar.gz
+    - CHROOT=xenial-test-chroot
+    - LC_ALL=C.UTF-8
+    - LANG=C.UTF-8
+    - SHCKOPTS="-e SC2013,SC2129"
+      # SC2013: we actually want words in the cases where this complaint happens
+      # SC2129: we want to see when the logging stopped so grouping is no option
+install:
+  - mkdir -p $CHROOT/build
+  - wget -q -O - $URL | zcat - | sudo tar x -C $CHROOT
+  - sudo cp /etc/resolv.conf $CHROOT/etc/
+  - sudo mount --bind /dev $CHROOT/dev
+  - sudo chroot $CHROOT apt update
+  - sudo chroot $CHROOT apt install -y shellcheck file build-essential devscripts
+  - sudo cp -a initramfs config $CHROOT/build/
+    # install all build deps defined in the control file
+  - sudo chroot $CHROOT apt install -y $(grep -h Build-Depends {initramfs,config}/debian/control|sed 's/^.*://g;s/,//g;s/([^)]*)//g'| tr -d '\n'| sort | uniq)
+script:
+  - sudo chroot $CHROOT sh -c 'cd build; for file in $(find . | xargs file | grep shell |grep -v .git| sed "s/:.*$/ /g" | tr -d "\n"); do shellcheck $SHCKOPTS $file; done'
+    # test build src packages unsigned in a xenial chroot
+  - sudo chroot xenial-test-chroot sh -c 'cd build/initramfs; LC_ALL=C.UTF-8 dpkg-buildpackage -rfakeroot -S -sa -us -uc'
```

Comment:
Why above uses `$CHROOT` but these don't?

The common prefix here might also be turned into a variable btw (`$CHROOT_RUN` or something).